examples/apps/cold_chain_eval/client/ColdChainAPI.py

```diff
@@ -0,0 +1,392 @@
+# Client for Coldchain workload
+import os
+from os import urandom, path, environ
+from flask import request
+from flask import Flask
+from avalon_client_sdk.quorum.quorum_worker_registry_client import QuorumWorkerRegistryImpl
```

Comment:
You need to run `bin/run_lint` and fix any formatting issues in this and other files.